src/01-functions.js

```diff
@@ -168,7 +190,9 @@ function joinStringsWithSpaces() {}
  *  getFirstAndLastCharacter("upwards");
  *  //> "us"
  */
-function getFirstAndLastCharacter() {}
+function getFirstAndLastCharacter(a) {
+  return a.charAt(0) + a.charAt(a.length-1)
```

Comment:
We could do this in a more straightforward way by simply using bracket notation.